### ISSUE TITLE
Gold progress tracker isn't working for me.

### ISSUE THREAD

**Issue by MACDARK007** [Epix37/Hearthstone-Deck-Tracker#845]
Played a few games but nothing changed. Remains in 0/0 - 0/100G

**Comment by Epix37** [comment]
Do recorded games have the correct region attached?

**Comment by Epix37** [comment]
Ah I think I see the issue. I'm guessing you're on US?

**Comment by MACDARK007** [comment]
Yes,it's recording my games correctly. Yes,i'm on US,but i'm from South America

**Comment by MACDARK007** [comment]
Just tried a game more to check than wasn't nothing wrong about the Tracker since earlier says i needed to restart hearthstone,it continues without tracking the gold or the wins.

**Comment by Epix37** [comment]
It's actually just not updating it for US period. Will be fixed with the next update.
